lua/nvim-tree/renderer/builder.lua

```diff
@@ -294,14 +296,21 @@ function Builder:build(tree)
   return self
 end
 
-local function format_root_name(root_cwd, modifier)
-  local base_root = utils.path_remove_trailing(vim.fn.fnamemodify(root_cwd, modifier))
-  return utils.path_join { base_root, ".." }
+local function format_root_name(root_cwd, root_label)
+  if type(root_label) == "function" then
+    label = root_label(root_cwd)
+    if type(label) == "string" then
```

Comment:
Thanks, looks good, I am only thinking whether we should notify user that returned value must be string and until it is default value will be used instead. `vim.notify_once()`?

Reply:
I just realized I forgot to add make the `label` variable `local` in line 301. Should I commit the fix?